site/source/docs/porting/connecting_cpp_and_javascript/Interacting-with-code.rst

```diff
@@ -21,6 +21,7 @@ JavaScript and compiled C or C++:
 - Call JavaScript functions from **C/C++**:
 
   - :ref:`Using emscripten_run_script() <interacting-with-code-call-javascript-from-native>`.
+  - :ref:`Using EM_JS() <interacting-with-code-call-javascript-from-native>` (faster).
   - :ref:`Using EM_ASM() <interacting-with-code-call-javascript-from-native>` (faster).
```

Comment:
It's probably worth putting in the docs (maybe very briefly here and a little more below) that EM_JS should be preferred over EM_ASM, and why (and maybe what circumstances, if any, would require EM_ASM)